cmd/router/main.go

```diff
@@ -317,6 +317,26 @@ var runCmd = &cobra.Command{
 			}
 		}()
 
+		/* initialize metadata */
```

Comment:
```suggestion
		// The router will execute the file "init_sql" before starting.
```

Reply:
no, this is both init sql and coodinator init cases here